cartography/data/jobs/analysis/keycloak_inheritance.json

```diff
@@ -0,0 +1,30 @@
+{
+  "statements": [
+    {
+        "__comment__": "Inherit group memberships from subgroups to parent groups",
+        "query": "MATCH (u:KeycloakUser)-[:MEMBER_OF]->(g:KeycloakGroup)-[:SUBGROUP_OF*1..5]->(pg:KeycloakGroup) MERGE (u)-[r:INHERITED_MEMBER_OF]->(pg) ON CREATE SET r.firstseen = $UPDATE_TAG SET r.lastupdated = $UPDATE_TAG",
```

Comment:
[question] also, does it make sense to scope this to the sub-resource, the scope realm? 

oh, i guess because this is in the analysis/ folder that it only runs at the very end and not once per subresource anyway so we're good.